.jenkins/infrastructure/docker/build_linux_docker_images.Jenkinsfile

```diff
@@ -19,13 +19,11 @@ pipeline {
         string(name: "INTERNAL_REPO", defaultValue: "https://oejenkinscidockerregistry.azurecr.io", description: "Url for internal Docker repository")
         string(name: "OECI_LIB_VERSION", defaultValue: 'master', description: 'Version of OE Libraries to use')
         string(name: "DEVKITS_URI", defaultValue: 'https://oejenkins.blob.core.windows.net/oejenkins/OE-CI-devkits-d1634ce8.tar.gz', description: "Uri for downloading the OECI Devkit")
-        booleanParam(name: "PUBLISH_DOCKER_HUB", defaultValue: false, description: "Publish container to OECITeam Docker Hub?")
```

Comment:
Do we want this to be removed?

Reply:
Yes, we shouldn't be publishing to Docker Hub anymore as we don't have an enterprise account under their new usage policies.

Reply:
Reverted this removal until I can be sure